milestone2.ts

```diff
@@ -163,9 +199,18 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) => {
+  let resPerson: Volunteer = data[0];
+  for(let i=0; i<data.length; i++){
+    if (data[i].city === "SLO" && data[i].age > 40) {
+      data[i].position = "staff";
+      return data[i];
+
+    }
+  }
+}; // Code here
```

Comment:
redo with data handling functions (make sure to handle the case where you don't find a volunteer with those criteria)

Reply:
I have just resubmitted with the data handling function corrections made.